src/ReactWrapper.js

```diff
@@ -288,6 +288,15 @@ export default class ReactWrapper {
   }
 
   /**
+   * Returns true if the component rendered nothing, i.e., null or undefined.
+   *
+   * @returns {boolean}
+   */
+  isNull() {
```

Comment:
I feel like this name too specifically implies that the component did `return null` in its `render` method, even though `return undefined` (either explicitly or implicitly) should also be fine.